presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -168,58 +168,73 @@ Configuration Properties
-                                                   ``hadoop``. Otherwise, it will be ignored.
-
-``iceberg.catalog.cached-catalog-num``             The number of Iceberg catalogs to cache. This property is     ``10``
-                                                   required if the ``iceberg.catalog.type`` is ``hadoop``.
-                                                   Otherwise, it will be ignored.
-
-``iceberg.hadoop.config.resources``                The path(s) for Hadoop configuration resources.
-                                                   ``Example: /etc/hadoop/conf/core-site.xml.`` This property
-                                                   is required if the iceberg.catalog.type is hadoop. Otherwise,
-                                                   it will be ignored.
-
-``iceberg.max-partitions-per-writer``              The Maximum number of partitions handled per writer.          ``100``
-
-``iceberg.minimum-assigned-split-weight``          A decimal value in the range (0, 1] is used as a minimum      ``0.05``
-                                                   for weights assigned to each split. A low value may improve
-                                                   performance on tables with small files. A higher value may
-                                                   improve performance for queries with highly skewed
-                                                   aggregations or joins.
-
-``iceberg.enable-merge-on-read-mode``              Enable reading base tables that use merge-on-read for          ``true``
-                                                   updates.
-
-``iceberg.delete-as-join-rewrite-enabled``         When enabled, equality delete row filtering is applied        ``true``
-                                                   as a join with the data of the equality delete files.
-================================================== ============================================================= ============
+======================================================= ============================================================= ============
+Property Name                                           Description                                                   Default
+======================================================= ============================================================= ============
+``hive.metastore.uri``                                  The URI(s) of the Hive metastore to connect to using the
+                                                        Thrift protocol. If multiple URIs are provided, the first
+                                                        URI is used by default, and the rest of the URIs are
+                                                        fallback metastores.
+                                                        Example: ``thrift://192.0.2.3:9083`` or
```

Comment:
```suggestion
                                                        
                                                        Examples: ``thrift://192.0.2.3:9083`` or
```
Suggest adding a line above to format the `Example:` sentence in the build the way it appears in the code here.